### DISCUSSION TITLE
Do you think including the imports together with the component code would be a good idea?

### DISCUSSION THREAD

**Discussion by javac9** [composablehorizons/compose-examples#10] [Ideas]
For some reason on-the-fly imports don't work in my Android Studio so an option to have the imports readily available would be a nice addition, especially when it comes to larger components.
Also, sometimes we tend to choose a wrong import which may mislead us thinking something is wrong with the component.

**Comment by alexstyl** [comment]
I like this idea. let me see what i can do
